assets/blocks/course-outline/single-line-input/index.js

```diff
@@ -11,30 +12,33 @@ import { ENTER } from '@wordpress/keycodes';
  * @param {Function} props.onChange  Change callback.
  * @param {Function} props.onKeyDown Keydown callback.
  */
-const SingleLineInput = ( { onChange, onKeyDown, ...props } ) => {
-	/**
-	 * Handle change.
-	 *
-	 * @param {string} value Change value.
-	 */
-	const handleChange = ( value ) => {
-		onChange( value.replace( /\n/g, '' ) );
-	};
+const SingleLineInput = forwardRef(
```

Comment:
Should we move this to a shared namespace?

Reply:
You could do this in #3976 to avoid conflicts.

Reply:
I thought I already did it in that one :) Will update it with the move.